### PR TITLE
project addition by [Niladri2003] #653

### PR DESCRIPTION
**Pull request by Niladri2003**
## Related Issue

Added a project for the issue - data: project addition by Niladri2003 #653

## Description

Responsive landing page for Ed-tech platform.

### CODE REVIEW

File: src/DB/projects.json [hunk 1]
```diff
@@ -1883,5 +1883,23 @@
...
+      "YouTube": "",
+      "Twitter": ""
+    },
+    "Projects": [
+      {
+        "link": "https://eduhub-1tyq.vercel.app/",
+        "title": "Eduhub",
+        "description": "Baisc Landing page for Online ed-tech platforms",
+        "tech": ["Javascript", "CSS", "HTML", "Tailwind CSS"]
+      }
```
Comment (priyankarpal) [inline src/DB/projects.json:1902]: hey @Niladri2003 you already have a profile in projects hut so you don't need to add the same code multiple times 
just add this sample code to your previous code

Comment (Niladri2003) [reply src/DB/projects.json]: Required changes done.

Comment (priyankarpal) [reply src/DB/projects.json]: 👏🏻

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**